.github/CODEOWNERS

```diff
@@ -23,7 +23,6 @@
 /.travis.yml         @coq/ci-maintainers
 /.gitlab-ci.yml      @coq/ci-maintainers
 /.github/workflows   @coq/ci-maintainers
-/azure-pipelines.yml @coq/ci-maintainers
```

Comment:
It could make sense to reintroduce the `@coq/windows-build-maintainers` team in the `CODEOWNERS` file for the new `dev/ci/platform` directory.

Reply:
Oh sure, it was an oversight by me.

Reply:
Done, thanks.